app/views/admin/dashboard/index.html.erb

```diff
@@ -12,6 +12,8 @@
         <%= "Created On: #{order.created_at.to_date}" %>
         <%= content_tag :br %>
         <%= "Users: " %><%= link_to order.name, "admin/users/#{order.user_id}" %>
+        <%= content_tag :br if order.status == "Packaged" %>
```

Comment:
I'm a little confused here. Are there two if statements for the same thing? `order.status == "Packaged"`

Or are they doing something different?

Reply:
There are two that do the same thing. 😬
I'll fix it and push up my revision.

Reply:
I replaced the guard statements with an if block.